### PR TITLE
Feature/1189 default src tgt

### PR DESCRIPTION
**Pull request by Michiel-s**
Implements AmpersandTarski/Ampersand#1189

Waiting for release of Ampersand compiler >v4.4.3

### CODE REVIEW

File: src/Ampersand/Core/Relation.php [hunk 1]
```diff
@@ -339,4 +356,33 @@ public function empty(): void
...
+    }
+
+    protected static function resolveDefaultValue(string $value): string
+    {
+        if (substr($value, 0, 5) === '{php}') {
+            $code = 'return('.substr($value, 5).');';
+            $value = (string) eval($code);
+        }
+        return $value;
+    }
```
Comment (hanjoosten) [inline src/Ampersand/Core/Relation.php:387]: This suggests that the php part will return exactly one atom. Is this intended? What if the function returns multiple atoms? Should this behaviour be documented for the modeller?

Comment (Michiel-s) [reply src/Ampersand/Core/Relation.php:387]: You're right. We discussed that the eval may return multiple values. I'll update the code.

**Comment by Michiel-s** [comment]
@hanjoosten could we release a new version of the compiler soon? So I can also update the Dockerfile in the prototype repo to take that version? As this is new feature I think we can move to v4.5.0

File: src/Ampersand/Core/Relation.php (hunk 1 above)
Comment (Michiel-s) [reply src/Ampersand/Core/Relation.php]: To reduce complexity in the code I propose to allow only a single scalar value to be returned by the evalphp function.  I've added a check for this with proper error message if this goes wrong. If multiple values are needed, this can be done by specifying multiple `EVALPHP` parts (see below). I think this is an edge case anyway.

```adl
RELATION projectStatus[Project*ProjectStatus] [UNI,TOT] DEFAULT TGT EVALPHP "test()" TGT EVALPHP "test2()"
```

**Review by hanjoosten — APPROVED** [review]
